policyengine_canada/parameters/gov/provinces/nb/tax/income/credits/tuition_amount/nb_tax_on_taxable_income_rate.yaml

```diff
@@ -0,0 +1,9 @@
+description: New Brunswick phases tuition credit amount for tax on taxable income at this rate.
+values:
+  2022-01-01: 0.094
+metadata:
+  unit: currency-CAD
+  label: New Brunswick tax on taxable income rate
+  reference:
+    - title: New Brunswick tuition and education amounts
+      href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5004-s11/5004-s11-22e.pdf#page=1
```

Comment:
could you add the comment stating that it is not mentioned in the NB income tax act